### DISCUSSION TITLE
Packaging Beta Snap and AUR packages

### DISCUSSION THREAD

**Discussion by coder3101** [cpeditor/cpeditor#935] [Development]
Alpha and stable releases are packaged and available on snap and aur. But there might be some users who will prefer to have an option to to install beta version since it's slightly more tested as compared to alpha and slightly ahead of stable release. 

@ouuan and @neko-para what are your opinions. Is there any CI infra upgrade we need to achieve this? Was there any specific reason that beta was not packaged for AUR originally?

**Comment by ouuan** [comment]
AUR doesn't accept beta packages.

**Reply by coder3101** [reply]
What's stopping to create a new package `cpeditor-beta` that conflicts `cpeditor-git` and `cpeditor`.

This is how [nvidia beta driver](https://aur.archlinux.org/packages/nvidia-beta/) shares with AUR

**Reply by ouuan** [reply]
You said that: https://github.com/cpeditor/cpeditor/issues/209#issuecomment-609011237

The official repository only has stable packages, so I took it for granted that AUR is the same, except for VCS packages. Maybe AUR does accept beta packages.

**Reply by coder3101** [reply]
I only noticed it few days back that nvidia beta was available. I wanted to install it's beta version and found it in AUR so thought same could be done for cpeditor.
